sos/report/plugins/ceph_iscsi.py

```diff
@@ -20,14 +20,26 @@ class CephISCSI(Plugin, RedHatPlugin, UbuntuPlugin):
     containers = ("rbd-target-api.*", "rbd-target-gw.*")
 
     def setup(self):
-        self.add_copy_spec([
-            "/etc/tcmu/tcmu.conf",
-            "/var/log/**/ceph-client.*.log",
-            "/var/log/**/rbd-target-api.log",
-            "/var/log/**/rbd-target-gw.log",
-            "/var/log/**/tcmu-runner.log",
-            "/var/log/tcmu-runner.log"
-        ])
+        all_logs = self.get_option("all_logs")
+
+        self.add_container_logs(["/etc/tcmu/tcmu.conf",])
```

Comment:
`add_container_logs()` is meant to get the output of, for example, `podman logs -t $container`.

If you want to get log files out of a specific container, you need to use the regular `add_copy_spec()` but use the `container` parameter to specify the name of the container to copy the file from.

Reply:
Understood, thanks, I have reverted that minor change so that file is extracted using `add_container_logs()`, as it was before.